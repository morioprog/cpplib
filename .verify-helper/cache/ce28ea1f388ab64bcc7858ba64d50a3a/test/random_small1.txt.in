1000 1
1 527
1 87
1 407
2
1 215
1 907
2
1 841
1 1
1 183
1 889
1 490
1 817
1 221
1 510
1 699
2
1 300
1 707
1 346
1 967
2
1 856
1 55
1 637
2
1 960
1 195
2
1 364
1 114
1 200
1 761
1 235
1 560
1 490
1 150
1 81
1 311
1 122
1 111
1 2
1 508
1 664
1 829
1 252
1 889
1 408
1 354
1 962
1 657
2
1 552
1 288
1 979
1 824
1 270
1 525
1 144
1 192
1 82
1 713
1 182
1 293
1 801
1 502
1 732
1 157
1 645
1 723
1 967
1 219
1 949
1 187
2
1 864
1 30
1 790
2
1 201
1 539
1 27
1 315
1 892
1 508
1 185
1 889
1 147
1 262
1 642
1 69
1 709
1 165
1 948
1 206
1 99
1 780
1 110
1 28
1 142
1 717
1 837
2
1 346
1 8
1 106
2
1 351
1 932
1 418
1 472
1 772
1 85
2
2
1 697
1 252
1 407
1 981
1 562
1 891
1 503
2
1 962
1 942
1 37
1 822
1 849
1 166
1 720
2
1 854
1 885
1 729
1 66
1 130
1 658
1 642
1 169
1 649
1 347
2
1 978
2
1 802
1 949
1 542
2
1 266
1 631
1 359
1 817
1 947
1 243
1 253
1 982
1 489
2
1 538
1 220
1 295
1 515
1 421
1 963
2
1 267
1 942
1 352
1 223
1 910
1 518
1 513
1 301
1 87
2
1 509
1 281
1 902
1 501
1 815
1 91
1 224
1 264
1 592
1 131
1 840
2
1 190
1 165
1 862
1 134
2
1 528
1 908
2
1 478
1 74
2
1 910
1 260
1 598
1 732
1 584
1 291
1 784
1 411
1 360
1 860
1 435
1 132
1 584
2
1 857
1 700
1 736
1 305
1 270
1 760
1 709
1 364
1 238
1 903
1 66
1 44
1 174
1 879
2
1 343
1 257
1 331
2
2
1 828
1 614
1 765
2
1 10
1 370
1 778
1 608
1 76
1 501
1 130
2
1 794
1 964
1 490
1 725
2
1 875
1 610
2
2
1 295
1 505
2
1 170
1 994
1 520
1 797
2
1 772
1 239
1 792
1 749
1 852
1 620
1 622
1 198
1 373
1 1
1 640
1 565
1 758
1 46
1 813
2
2
1 391
1 995
1 86
2
1 966
1 275
1 557
1 977
1 649
1 83
1 945
1 31
1 848
1 579
1 748
1 196
1 223
1 2
1 328
1 447
1 722
1 196
1 201
1 673
1 548
1 483
1 631
1 736
1 485
1 133
1 570
2
1 237
2
1 238
1 760
1 17
1 849
1 51
1 445
1 178
2
1 169
1 948
1 831
2
1 342
1 563
2
1 669
2
1 724
1 939
1 647
1 77
1 925
1 438
1 136
1 960
1 255
1 435
1 96
1 451
1 178
1 500
1 862
1 434
1 353
2
1 381
1 681
1 134
1 306
1 167
1 123
1 340
1 795
1 694
1 623
1 12
1 265
2
1 72
1 240
1 211
1 725
1 660
1 499
2
1 153
1 368
1 948
1 618
1 88
1 216
1 417
1 518
1 735
1 190
1 103
2
1 716
1 747
1 199
1 120
1 420
1 335
2
1 640
1 50
1 836
1 460
1 307
2
1 70
1 214
1 362
1 712
1 433
1 802
1 901
1 733
2
1 192
1 103
1 487
1 160
1 780
2
1 23
1 711
1 437
1 626
1 44
2
1 483
2
1 494
2
1 467
1 559
1 201
1 995
1 205
1 688
1 778
1 476
1 185
1 290
1 465
1 155
1 911
1 306
1 407
1 459
1 938
1 393
1 124
1 47
1 849
1 68
1 268
1 188
1 250
1 307
1 930
1 184
1 191
2
1 142
1 499
1 323
2
1 119
1 91
1 332
1 846
1 107
1 845
1 197
2
1 645
1 219
2
1 570
1 192
2
1 795
1 934
1 373
1 74
1 159
1 387
1 434
1 808
1 713
1 338
1 57
1 282
1 507
1 883
1 496
1 677
2
1 429
1 110
1 717
1 22
1 232
1 781
1 763
1 977
1 299
1 915
1 669
1 606
1 415
1 452
1 467
1 402
1 604
1 876
1 829
1 312
1 696
1 873
1 209
2
1 724
2
1 885
1 159
2
1 130
1 544
1 902
1 616
1 657
1 47
1 457
1 902
1 3
1 836
2
1 272
1 833
2
1 632
1 347
1 863
1 949
1 749
1 504
1 828
1 466
1 437
1 304
1 481
2
1 422
1 13
1 804
1 429
1 994
1 132
1 754
1 143
1 437
2
1 778
1 950
1 690
1 339
2
1 100
1 891
1 683
1 105
1 56
1 619
1 708
1 396
1 287
1 464
2
1 479
1 225
1 779
1 74
1 890
1 5
1 250
1 744
1 371
1 785
1 465
1 431
1 95
1 390
1 608
1 755
1 551
1 998
1 96
1 289
1 515
1 467
1 546
1 586
1 508
1 525
1 320
1 12
1 0
1 212
1 777
1 654
1 621
1 188
1 784
1 274
1 694
1 386
1 684
2
1 563
1 909
1 764
1 359
2
2
1 980
1 671
1 538
1 985
1 566
1 794
1 438
1 682
1 928
1 125
1 260
1 304
1 603
1 55
1 133
1 528
1 194
1 200
1 372
1 764
1 287
1 18
2
1 307
1 581
1 714
1 779
2
1 267
1 980
1 169
1 536
1 252
1 896
2
1 996
1 270
1 304
1 215
1 976
1 944
1 40
1 965
1 366
1 818
1 584
1 437
1 951
1 124
1 263
1 987
1 199
1 534
1 309
1 574
1 209
1 223
1 168
1 663
1 325
1 312
1 718
1 30
1 741
1 987
1 633
1 341
2
1 33
1 436
2
1 787
1 642
1 270
1 720
1 395
1 352
1 568
2
1 741
1 958
1 278
1 563
2
1 880
1 178
1 410
1 775
1 815
1 668
2
1 326
2
1 325
1 690
1 994
2
1 830
2
1 324
1 757
1 844
1 210
1 958
2
1 727
1 394
1 522
1 72
1 281
1 357
1 11
1 693
1 374
1 998
1 702
1 412
1 712
1 498
2
1 98
1 843
1 8
1 915
1 878
1 760
1 681
1 364
2
1 842
2
1 957
1 183
2
1 438
1 724
1 570
1 405
1 937
2
1 729
1 39
1 581
1 129
1 475
1 698
1 709
1 914
1 293
1 261
1 954
1 56
1 698
1 239
1 488
1 618
1 938
1 456
2
1 242
1 483
1 107
2
1 605
1 906
2
1 250
1 107
1 601
1 280
1 666
1 551
1 280
2
1 201
1 967
1 862
1 323
1 271
1 666
1 580
1 860
1 148
1 896
1 484
1 301
1 515
1 77
1 950
1 431
1 313
1 428
1 481
1 688
1 958
1 536
1 335
1 271
1 35
1 416
1 602
1 580
1 298
1 31
1 435
1 950
1 768
1 697
1 151
1 947
1 135
2
1 993
1 950
1 856
2
2
2
1 654
1 464
1 42
1 725
1 503
1 620
1 272
1 761
1 859
1 918
1 571
1 914
1 451
1 271
1 621
1 880
1 283
1 175
1 302
1 238
1 931
2
2
1 886
2
1 58
1 502
1 643
1 302
1 800
1 790
2
1 132
1 950
1 198
1 425
1 688
2
1 316
1 447
1 278
1 951
1 559
1 171
1 461
2
1 216
1 473
1 492
1 211
1 569
1 779
1 55
1 628
1 678
1 821
1 872
1 403
2
1 17
1 982
1 308
1 305
1 310
2
1 567
2
1 574
1 507
1 432
1 178
1 603
1 353
1 581
1 903
1 552
1 470
1 682
1 864
1 63
1 628
2
1 353
1 9
1 576
1 187
1 722
1 799
1 994
1 965
1 538
1 684
1 781
1 174
1 388
1 253
1 699
1 337
1 620
1 454
1 697
1 255
1 634
2
1 584
2
1 544
1 673
1 93
1 29
1 127
1 212
1 237
1 346
1 227
1 684
1 550
1 701
1 598
1 456
1 846
1 80
1 147
1 859
1 250
1 214
1 325
2
1 299
1 750
1 749
1 847
1 126
1 840
2
1 464
1 586
1 803
2
1 604
1 768
1 806
1 167
1 556
1 174
1 670
1 549
1 185
1 126
1 698
1 481
1 366
1 927
1 310
1 350
1 940
1 365
1 133
2
1 201
1 18
1 920
1 329
1 931
1 593
1 516
1 860
2
1 168
1 410
1 260
1 162
1 300
