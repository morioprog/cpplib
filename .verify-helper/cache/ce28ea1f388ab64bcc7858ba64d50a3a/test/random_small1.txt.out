87
215
1
183
55
195
2
81
30
27
8
28
69
82
37
66
85
99
106
110
87
91
111
114
74
122
44
66
130
131
10
76
130
132
134
142
1
46
86
2
31
17
51
83
133
77
12
72
88
96
50
70
103
23
44
103
47
68
91
107
119
57
22
74
110
3
47
120
13
123
56
0
5
12
18
55
74
30
33
40
95
96
100
105
124
124
11
8
72
98
125
39
56
107
107
31
35
77
129
42
130
132
58
132
133
55
17
134
63
9
135
29
80
93
126
18
